packages/app-bridge/src/react/useBulkDownload.ts

```diff
@@ -0,0 +1,75 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import type { AppBridgeBlock } from '../AppBridgeBlock';
+
+export enum BulkDownloadState {
+    Init = 'init',
+    Started = 'started',
+    Pending = 'pending',
+    Ready = 'ready',
+    Error = 'error',
+}
+
+export const useBulkDownload = (appBridge: AppBridgeBlock) => {
+    const [status, setStatus] = useState<BulkDownloadState>(BulkDownloadState.Init);
+    const [token, setToken] = useState<string>('');
+    const [signature, setSignature] = useState<string>('');
+    const [download, setDownload] = useState<string>('');
+
+    const generateBulkDownload = async (assetIds: number[], setIds?: number[]) => {
+        try {
+            const token = await appBridge.getBulkDownloadToken(assetIds, setIds ?? []);
```

Comment:
```suggestion
    const generateBulkDownload = async (assetIds: number[], setIds?: number[]) => {
        try {
            const token = await appBridge.getBulkDownloadToken(assetIds, setIds);
```
appBridge.getBulkDownloadToken has the same function signature